engine/gamesys/src/gamesys/components/comp_sprite.cpp

```diff
@@ -1844,6 +1942,8 @@ namespace dmGameSystem
          */
 
         SpriteWorld* world = (SpriteWorld*)params.m_World;
+        UpdateCache(world);
```

Comment:
Perhaps rename to UpdateAnimationDataCache() ?

Reply:
Renamed